xml/System.Windows.Controls/StickyNoteControl.xml

```diff
@@ -665,9 +665,10 @@
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- The  identifier for the <xref:System.Windows.Controls.StickyNoteControl.IsExpanded%2A> dependency property.  
```

Comment:
Was there a dropped word/link here?

Reply:
No. I thought perhaps there was, but this is a standard remark for other StickyNoteControl dependency properties.